### PR TITLE
feat: add signup

### PR DESCRIPTION
**Pull request by AlexisToribio**

### CODE REVIEW

File: components/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+type Props = {
+  children: React.ReactNode;
+  toggleShow: boolean;
+  setToggleShow: (toggleShow: boolean) => void;
+};
+
+function Modal({ children, toggleShow, setToggleShow }: Props) {
+  return (
+    <>
+      {toggleShow && (
```
Comment (rigomart) [inline components/Modal.tsx:13]: You should make the conditional rendering where the Modal component is used (in index.tsx aparently)

File: config/axios.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const instance = axios.create({
+  baseURL: 'https://twitter-clone-pts.herokuapp.com',
```
Comment (rigomart) [inline config/axios.ts:4]: Add Authorization header later

File: pages/signup.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+    axios
+      .post('/auth/register', {
+        name,
+        email,
+        password,
+        b_date,
+      })
+      .then((res) => {
+        console.log(res);
```
Comment (rigomart) [inline pages/signup.tsx:45]: Give feedback to the user when the data is send, whether there are errors or the request was succesful

**Review by rigomart — COMMENTED** [review]
- Give feedback to the users when signing up.
- Change the conditional rendering for the Modal component.
- Add headers for later

**Comment by AlexisToribio** [comment]
Ok man. I will work on it.

**Pull request merged**